vm-superio/src/serial.rs

```diff
@@ -289,6 +291,18 @@ pub enum Error<E> {
     FullFifo,
 }
 
+impl fmt::Display for Error<io::Error> {
```

Comment:
Mh, not sure if `E` is ever not `io::Error`, but could we still do 
```suggestion
impl<E: fmt::Display> fmt::Display for Error<E> {
```
here?

Reply:
got it!